src/store/slices/node/initialState.ts

```
@@ -67,10 +66,7 @@
     data: AddressesType;
     isFetching: boolean;
   };
-  aliases: {
-    data: GetAliasesResponseType | null;
-    isFetching: boolean;
-  };
+  aliases: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)